### ISSUE TITLE
Deprecated functions reported when compilinig against libssh master

### ISSUE THREAD

**Issue by adelton** [ParallelSSH/ssh-python#7]
Hello,

when building ssh-python against libssh master, deprecated functions are reported.

In the https://github.com/ParallelSSH/ssh-python checkout directory I do

* `rm -rf libssh && git clone git://git.libssh.org/projects/libssh.git --depth 1`
* `python3 setup.py build_ext --inplace`

In the output, I see
```
skipping 'ssh/session.c' Cython extension (up-to-date)
building 'ssh.session' extension
gcc -pthread -Wno-unused-result -Wsign-compare -DDYNAMIC_ANNOTATIONS_ENABLED=1 -DNDEBUG -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fexceptions -fstack-protector-strong -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -D_GNU_SOURCE -fPIC -fwrapv -fPIC -Ilibssh/include -I/usr/include/python3.7m -c ssh/session.c -o build/temp.linux-x86_64-3.7/ssh/session.o -O3
ssh/session.c: In function ‘__pyx_pf_3ssh_7session_7Session_46is_server_known’:
ssh/session.c:4782:9: warning: ‘ssh_is_server_known’ is deprecated [-Wdeprecated-declarations]
         __pyx_v_rc = ssh_is_server_known(__pyx_v_self->_session);
         ^~~~~~~~~~
In file included from ssh/session.c:569:
libssh/include/libssh/libssh.h:551:31: note: declared here
 SSH_DEPRECATED LIBSSH_API int ssh_is_server_known(ssh_session session);
                               ^~~~~~~~~~~~~~~~~~~
ssh/session.c: In function ‘__pyx_pf_3ssh_7session_7Session_118write_knownhost’:
ssh/session.c:9577:9: warning: ‘ssh_write_knownhost’ is deprecated [-Wdeprecated-declarations]
         __pyx_v_rc = ssh_write_knownhost(__pyx_v_self->_session);
         ^~~~~~~~~~
In file included from ssh/session.c:569:
libssh/include/libssh/libssh.h:549:31: note: declared here
 SSH_DEPRECATED LIBSSH_API int ssh_write_knownhost(ssh_session session);
                               ^~~~~~~~~~~~~~~~~~~
ssh/session.c: In function ‘__pyx_pf_3ssh_7session_7Session_120dump_knownhost’:
ssh/session.c:9686:9: warning: ‘ssh_dump_knownhost’ is deprecated [-Wdeprecated-declarations]
         __pyx_v__known_host = ssh_dump_knownhost(__pyx_v_self->_session);
         ^~~~~~~~~~~~~~~~~~~
In file included from ssh/session.c:569:
libssh/include/libssh/libssh.h:550:33: note: declared here
 SSH_DEPRECATED LIBSSH_API char *ssh_dump_knownhost(ssh_session session);
                                 ^~~~~~~~~~~~~~~~~~
gcc -pthread -shared -Wl,-z,relro -Wl,-z,now -specs=/usr/lib/rpm/redhat/redhat-hardened-ld -g build/temp.linux-x86_64-3.7/ssh/session.o -L/root/ssh-python/src/src -L/usr/lib64 -Wl,--enable-new-dtags,-R$ORIGIN/. -lssh -lpython3.7m -o /root/ssh-python/ssh/session.cpython-37m-x86_64-linux-gnu.so
```

**Comment by adelton** [comment]
This is on Fedora 29 with gcc-8.3.1-2.fc29.x86_64.

**Comment by pkittenis** [comment]
Hi there,

Thanks, looks like those function bindings can be removed from session.pyx. PR is welcome.

**Comment by adelton** [comment]
Filed https://github.com/ParallelSSH/ssh-python/pull/10.

**Comment by pkittenis** [comment]
Resolved at #10.
